### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by ishanchawla1**
This assignment consists of making a clone of a Apple.com page clone.

We can find the original project specification at https://www.theodinproject.com/courses/html5-and-css3/lessons/building-with-backgrounds-and-gradients

Here we are using positioning techniques such as Flexbox or Grid and Float to create the page.

### CODE REVIEW

**Review by rOluochKe — CHANGES_REQUESTED** [review]
Hello @ishanchawla1 
###  Good job for the work done so far :clap: however, there are a few issues you need to implement before your project could be approved.
- Remove the horizontal scroll bar
- The hero background is not displaying
image nil
### Make these changes and resubmit for another code review. Good luck.
- Happy coding!

File: index.html [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+                <ul class="table">
+                    <li class="listItem"><a href="#" class="noStyle1"> Site Map </a> </li>
+                    <li class="listItem"><a href="#" class="noStyle1"> Hot News </a> </li>
+                    <li class="listItem"><a href="#" class="noStyle1"> RSS Feeds</a> </li>
+                    <li class="listItem"><a href="#" class="noStyle1"> Media Info </a> </li>
+                    <li class="listItem"><a href="#" class="noStyle1"> Environment </a> </li>
+                    <li class="listItem"><a href="#" class="noStyle1"> Job Opportunity </a> </li>
+                    <li class="listItem"><a href="#" class="noStyle1"> Contact Us </a> </li>
+                    <li class="image"><a href="#" class="noStyle1"> 
+                        <img class="footerFlag" src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/global/elements/flags/16x16/usa_2x.png" style="width: 15px;" alt="flag">
```
Comment (euqueme) [inline index.html]: remove style="width: 15px;" on line #89 all style should be declared in the CSS file

**Review by euqueme — CHANGES_REQUESTED** [review]
> ### Pull Request under the review by Maria Eugenia Queme P. (TSE)

### Hello @ishanchawla1 :handshake: 

### Thank you for doing the changes requested by the previous reviewer :medal_sports:  your project looks professional :necktie:  and your page is very similar to the original, :microscope:  you only have to do a couple of changes to move on :point_down: 

## Section Specific

#### no inline styling

- [ ] I have found inline styling in your code on line #89 please fix it

#### no inefficient selectors

- [ ] You've use these selectors several times within your CSS file, let's chain declare them or make independent classes to keep your code DRY (Don't Repeat Yourself)

- margin-left: auto; (4 times)
- margin-right: auto; (4 times)
- display: block; (3 times)
- position: relative; (2 times)
- display: flex; (6 times)
- flex-wrap: wrap; (6 times)
- height: 36px; (2 times)
- text-align: center; (2 times)
- color: white; (4 times)
- padding: 0%; (2 times)
- text-decoration: none; (3 times)
- color: #08c; (2 times )
- font-size: 12px; (3 times)
- list-style: none; (5 times)
- display: table-cell; (2 times)
- margin-top: 0%; (2 times)
- max-width: 1440px; (2 times)
- width: 1440px; (3 times)
- text-decoration: underline; (3 times)

[OPTIONAL]

- [ ] I strongly encourage the use of the [`<main>` tag](https://www.w3schools.com/tags/tag_main.asp) for the dominant content of the `<body>` of a document

### That's it! :raised_hands: 

### Please :pray: do the changes :man_technologist: and submit back for a code review when you are done :memo: 

Happy Coding! :rocket: 

Any question? ask me on slack **@ Maru** :woman_technologist: 

PROJECT STATUS: REQUESTED CHANGES :vertical_traffic_light:

**Review by nikasulo — APPROVED** [review]
# Great work @ishanchawla1  !
Please merge and keep moving forward, see you at the top! :rocket: 

#### [OPTIONAL]
Please consider comparing your site's design with the original's and try to improve it, consider the navigation bar especially

#### Horizontal Scroll
You seem to have a little bug with one of the elements of the page, shooting out beyond the browser's maximum width. This may be as a result of a margin or padding you have set up. One of the easiest ways I have found to correct this issue is to set the default `box-sizing` property to `border-box` as follows `*{box-sizing: border-box}`. Please read some more about it [here](https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing).


##### Think I missed something? Need some help with this or any of your projects? Please feel free to reach out to me on [Twitter](https://twitter.com/oluwadamilareo_) or [Instagram](https://instagram.com/oluwadamilare_olusakin), even if you just want to chat and I will respond to you as soon as I can.

**Pull request merged**